docs/guides/basic-compositions/build-composition.md

```diff
@@ -0,0 +1,279 @@
+         - type: map
+           map:
+             small: e2-standard-2
+             medium: e2-standard-4
+             large: e2-standard-8
+```
+
+Now to explain what you just wrote:
+
+* You have two patches, one for each field exposed in the XRD. 
+* The fields exposed to the end user are string enums, and the values do not directly map to the values needed by the Managed Resource, so we set up a transform to mutate the value inputted by users to the value required by the Managed Resource. Because this is a Compute Engine instance, you can find the values GCP uses for [region](https://cloud.google.com/compute/docs/regions-zones) and [machineType](https://cloud.google.com/compute/docs/general-purpose-machines#e2_machine_types) respectively.
+
+### Compose the Network objects
+
+You will notice under the `networkInterface` object, there are two "selector" fields: `networkSelector` and `subnetworkSelector`. When GCP creates a new Compute Engine instance, it also creates network obejcts for each instance. Therefore, you must do the same in Crossplane. These objects, `Network` and `Subnetwork` are _also_ Managed Resources. Crossplane is using label matching to establish a relationship between these MRs--the `Instance` to each networking MR.
+
+Therefore, you need two add two additional composed Managed Resources to this Composition: a `Network` MR and a `Subnetwork` MR. Just as you did for the `Instance` MR, you can look these up on the Marketplace to find which fields are required when creating these objects, look at examples, etc. Let's first add the `Network` as another object under the `resources` array:
+
+```yaml
+...
+    - name: network-instance
+      base:
+        apiVersion: compute.gcp.upbound.io/v1beta1
+        kind: Network
+        metadata:
+          labels:
+            network-name: network-endpoint
+          name: network-endpoint
+        spec:
+          forProvider:
+            autoCreateSubnetworks: false
+```
+
+And now let's add the `Subnetwork` as another object, also under the `resources` array:
+
+```yaml
+...
+    - name: subnet-instance
+      base:
+        apiVersion: compute.gcp.upbound.io/v1beta1
+        kind: Subnetwork
+        metadata:
+          labels:
+            subnet-name: network-endpoint
+          name: network-endpoint
+        spec:
+          forProvider:
+            ipCidrRange: 10.2.0.0/16
+            networkSelector:
+              matchLabels:
+                network-name: network-endpoint
+            region: us-west1
+      patches:
+        - fromFieldPath: "spec.parameters.region"
+          toFieldPath: "spec.forProvider.region"
+          transforms:
+          - type: map
+            map:
+              east: us-east1
+              west: us-west1
+```
+
+Now to explain what you just wrote:
+
+* You defined two additional MRs as part of this Composition. In total, this Composition composes 3 Managed Resources: an `Instance`, a `Network`, and a `Subnetwork`.

```

Comment:
You may be interested in a new feature I just added called "hover highlight" 
https://github.com/crossplane/docs/pull/238

I'll have docs for the feature shortly.